config/boards/mixtile-blade3.csc

```diff
@@ -11,17 +11,11 @@ declare -g BOOTCONFIG="blade3_defconfig"
 declare -g IMAGE_PARTITION_TABLE="gpt"
 declare -g UEFI_EDK2_BOARD_ID="blade3" # This _only_ used for uefi-edk2-rk3588 extension
 
-# newer blobs from rockchip. tested to work.
-# set as variables, early, so they're picked up by `prepare_boot_configuration()`
-# @TODO: consider removing those, as the defaults in rockchip64_common have been bumped up
-declare -g DDR_BLOB='rk35/rk3588_ddr_lp4_2112MHz_lp5_2736MHz_v1.11.bin'
-declare -g BL31_BLOB='rk35/rk3588_bl31_v1.38.elf'
-
 # post_family_config hook which only runs when branch is legacy.
 function post_family_config__uboot_mixtile() {
 	display_alert "$BOARD" "Configuring Mixtile u-boot" "info"
 	declare -g BOOTSOURCE='https://github.com/radxa/u-boot.git'
-	declare -g BOOTBRANCH="commit:ddc91cd08c10f625f7a7c93033042aa4071c78a8" # specific commit in next-dev branch
+	declare -g BOOTBRANCH='branch:next-dev-v2024.03'
 	declare -g OVERLAY_PREFIX='rockchip-rk3588'
 	declare -g BOOTDIR="u-boot-${BOARD}"                   # do not share u-boot directory
 	declare -g BOOTPATCHDIR="legacy/u-boot-mixtile-rk3588" # Few patches in there; defconfig & PD hacks
```

Comment:
I've just tested on-device on this incarnation. Since it's approved I'm gonna merge, and later when I have time I'm gonna de-null-patch and join the family...